pkg/util/webhooks/tls.go

```diff
@@ -29,3 +69,135 @@ func SetupTLS(caManager ClientCAManager, serverCert tls.Certificate, clientAuth
 	tlsConfig.BuildNameToCertificate()
 	return tlsConfig
 }
+
+func SetupTLSForVirtHandlerServer(certManager bootstrap.CertificateManager) *tls.Config {
```

Comment:
any way to unit test these setup functions and validate tls configs? maybe we're already doing that?

Reply:
now unit-tested.